UnoCpp/UnoCpp/UnoCpp.cpp

```diff
@@ -1,10 +1,23 @@
 #include <iostream>
 
+#include "ConsoleIO.h"
 #include "GameManager.h"
 
 int main()
 {
 	GameManager gameManager{};
 	gameManager.Setup();
+	ConsoleIO::GetInput<int>("");
 	return 0;
 }
+
+// int Core::RandomRange(int min, int maxInclusive)
+// {
+// 	std::mt19937 mt{std::random_device{}()};
+// 	std::uniform_int_distribution distribution = std::uniform_int_distribution{min, maxInclusive};
+//
+// 	return distribution(mt);
+// }
+
+// const unsigned seed = static_cast<unsigned>(std::chrono::system_clock::now().time_since_epoch().count());
+// std::ranges::shuffle(cards, std::default_random_engine(seed));
```

Comment:
Esse bloco todo comentado pode ser removido?

Reply:
é pra eu consultar se precisar, dps apago

Reply:
Não tem problema em deixar pra consulta (depois vai apagar mesmo...), mas uma ideia que faço é apagar e consultar no histórico de commits. Se for algo mais importante, vale marcar o commit com uma tag (não acho que esse aqui seja o caso).